types/codec.go

```diff
@@ -10,9 +10,10 @@ func init() {
 	RegisterCodec(typesCodec)
 }
 
-const (
+var (
+	appName = "emint"
 	// Amino encoding name
-	EthermintAccountName = "emint/Account"
+	EthermintAccountName = appName + "/Account"
```

Comment:
To be safe, probably should keep the codec registered name the same. Might not cause conflicts between differently named clients.